decidim-bulletin_board-ruby/lib/decidim/bulletin_board/authority/get_election_results.rb

```diff
@@ -37,14 +37,24 @@ def call
             end
           end
 
-          broadcast(:ok, response.data.election.log_entries)
+          @signed_data = response.data.election.log_entries.first.signed_data
```

Comment:
Genuine question out of ignorance:

In case we called this command when the election does NOT yet have the `end_tally` log entry, would the above GraphQL query raise an error that will be rescued or will we get an empty `response.data.election.log_entries` array and fail with an uncaught exception here?

Reply:
Thanks a lot for that!!